go.mod

```diff
@@ -453,3 +453,6 @@ replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday
 //
 // Then, run `make modtidy` in this repository.
 // This ensures that go.mod and go.sum are up-to-date.
+
+// TODO: REMOVE WHEN https://github.com/dapr/components-contrib/pull/3178 IS MERGED
+replace github.com/dapr/components-contrib => github.com/italypaleale/dapr-components-contrib v1.10.2-0.20231015174010-5746c8f67dcc
```

Comment:
https://github.com/dapr/components-contrib/pull/3178 is merged, lets remove the replace command.

Reply:
Sorry, I forgot to "git push"! Fixing now